### ISSUE TITLE
KB should no longer attempt to modify VARS database directly. 

### ISSUE THREAD

**Issue by hohonuuli** [mbari-media-management/vars-kb#1]
The kb app was designed to interact directly with the annotations database. Changes to concepts would be propagated directly to the annotations that use that concept. 

The new mechanism is to use an API endpoint in M3_ANNOTATIONS to make the database change. Need to decide if VARS will issue call through AMQP or call the endpoint directly.

**Comment by hohonuuli** [comment]
This issue was prompted by [vars-annotation #111](https://github.com/mbari-media-management/vars-annotation/issues/111)

**Comment by hohonuuli** [comment]
I've created a new build of the vars-kb using this repo and deployed it internally at MBARI. This build no longer attempts to modify the annotations directly.

**Comment by NancyJS** [comment]
Forgive me, not sure where to report this. I am using the new build delivered March 7. When I approved the deletion of Alvinellinae, I was surprised the KB gave me this error because I thought this feature was temporarily disabled, per above. Also, using the new Query FX, I find 0 returns of Alvinellinae, so not sure where the KB is finding this lonely 1 Alvinellinae?
 
Thanks.
~njs

<img width="740" alt="Screen Shot 2019-03-13 at 9 43 37 AM" src="https://user-images.githubusercontent.com/19783409/54298447-00007c00-4576-11e9-9086-d66ad81e9159.png">

**Comment by hohonuuli** [comment]
Looking into it.

**Comment by hohonuuli** [comment]
__Sanity check 1__: Executing

```sql
SELECT 
  *
FROM
  annotations
WHERE 
  concept = 'Alvienellinae'
```

return no annotations.

__Sanity check 2__: Executing

```sql
SELECT 
  *
FROM
  observations
WHERE 
  concept = 'Alvienellinae'
```

returns no observations

**Comment by hohonuuli** [comment]
Calling <http://m3.shore.mbari.org/anno/v1/observations/concept/count/Alvinellinae> returns `{"concept":"Alvinellinae", "count":"0"}`

**Comment by NancyJS** [comment]
Thanks for double-checking!

Update: I just deleted the concept "Alvinellinae" in the KB and did NOT get the same error I previously attached. Maybe you just fixed this, Brian, but why did it tell us something different the first time?

**Comment by hohonuuli** [comment]
> why did it tell us something different the first time?

I have no idea! Were you running the old VARS KB app??

Just a note that I'm in the process of modifying the KB app to make changes via the annosaurus web service. I've made the code changes and hope to test it later today.

**Comment by hohonuuli** [comment]
Fixed in release 11.0.4
